### DISCUSSION TITLE
How To Validate Media In Blocks

### DISCUSSION THREAD

**Discussion by freshleafmedia** [area17/twill#1642] [Q&A]
When adding validation to a block how do you validate that, for example, a block must have media attached.

This is what I expected to work:

```blade
@twillBlockTitle('Image')
@twillBlockIcon('image')

@twillBlockValidationRules([
    'image' => [
        'required',
    ],
])

@twillBlockValidationRulesForTranslatedFields([])

@formField('medias', [
    'name' => 'image',
    'label' => 'Image',
    'required' => true,
])
```

But it does not, you just get a validation error "image field is required"

**Comment by haringsrob** [comment]
Hey @freshleafmedia,

Thank you for reporting this issue.

I see now that for media's browsers, relations the error keys are not correctly set. I will work on a fix for this.

**Comment by haringsrob** [comment]
Hey @freshleafmedia

We have prepared a fix that will be in the next release.

https://github.com/area17/twill/pull/1648

Cheers,
Rob

**Reply by freshleafmedia** [reply]
Hi @haringsrob

Awesome thanks, I'll give it a test
